docs/tutorials/getting_started.rst

```diff
@@ -2,46 +2,89 @@
 
 .. include:: ../tutorials/glossary_substitutions.rst
 
-***************
-Getting Started
-***************
+*****************************
+Getting Started with Baseband
+*****************************
 
-For most file formats, one can simply import baseband and use `baseband.open`
-to access the file.  This gives one a filehandle from which one can read
-decoded samples::
+This quickstart tutorial is meant to help the reader hit the ground running
+with Baseband.  All topics here are covered in more detail in :ref:`Using
+Baseband <using_baseband>`, which additionally discusses writing to files.
 
-    >>> import baseband
-    >>> from baseband.data import SAMPLE_DADA
-    >>> fh = baseband.open(SAMPLE_DADA)
-    >>> fh.read(3)
-    array([[ -38.-38.j,  -38.-38.j],
-           [ -38.-38.j,  -40. +0.j],
-           [-105.+60.j,   85.-15.j]], dtype=complex64)
-    >>> fh.close()
-
-For other file formats, a bit more information is needed.  Below, we cover the
-basics of :ref:`inspecting files <getting_started_inspecting>`, :ref:`reading
-<getting_started_reading>` from and :ref:`writing <getting_started_writing>`
-to files, and :ref:`converting <getting_started_converting>` from one format
-to another.  We assume that Baseband as well as `NumPy
-<http://www.numpy.org/>`_ and the `Astropy`_ units module have been imported::
+To install Baseband, please see :ref:`Installing Baseband <install_baseband>`.
+We recommend installing using `pip <http://www.pip-installer.org/en/latest/>`_
+within a `virtual environment <https://virtualenv.pypa.io/en/stable/userguide/>`_.
+
+.. _getting_started_opening:
+
+Opening Files
+=============
+
+When using Baseband, we typically will also use `numpy` and the `astropy.units`
```

Comment:
Can we move this to the section above? I.e., that presumes one has installed baseband and imported these things. Keep it short...

Reply:
Moved.